build.gradle

```diff
@@ -35,8 +35,24 @@ buildscript {
         bwcFlowFrameworkPath = bwcFilePath + "flowframework/"
 
         isSameMajorVersion = opensearch_version.split("\\.")[0] == bwcVersionShort.split("\\.")[0]
+
+        // Version constants for dependencies
+        awsEncryptionSdkVersion = "3.0.1"
+        awsCryptoMaterialProvidersVersion = "1.10.1"
+        dafnyRuntimeVersion = "4.10.0"
+        smithyDafnyVersion = "0.1.1"
+        gsonVersion = "2.13.1"
+        junitJupiterVersion = "5.12.2"
+        jakartaJsonBindVersion = "3.0.1"
+        jakartaJsonVersion = "2.0.1"
+        yassonVersion = "3.0.4"
+        parssonVersion = "1.1.7"
         swaggerVersion = "2.1.26"
         swaggerCoreVersion = "2.2.30"
+        httpclient5Version = "5.4.4"
```

Comment:
can we use version from core: https://github.com/opensearch-project/OpenSearch/blob/main/gradle/libs.versions.toml#L45